_sass/_publications.scss

```diff
@@ -5,7 +5,7 @@
   
   .pill {
     display: inline-block;
-    width: 100px;
+    width: 25%;
```

Comment:
I think one of the screenshots in the PR description shows it on the
smallest size

On Sunday, July 5, 2015, David Paquette <notifications@github.com> wrote:

> In _sass/_publications.scss
> <https://github.com/westerndevs/western-devs-website/pull/48#discussion_r33894247>
> :
>
> > @@ -5,7 +5,7 @@
> >
> >    .pill {
> >      display: inline-block;
> > -    width: 100px;
> > +    width: 25%;
>
> How does this look on a small screen / phone?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/westerndevs/western-devs-website/pull/48/files#r33894247>
> .
>